tests/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+enable_testing()
+find_package(GTest 1.10 REQUIRED)
+
+# Unit tests
+add_executable(cps-tests loader.cpp utils.cpp version.cpp)
```

Comment:
```suggestion
add_executable(cps-tests
  loader.cpp
  utils.cpp
  version.cpp
)
```

Reply:
Looking at the commit history, I think this repo has been using rebasing to incorporate changes. So I'll try to keep the commit history clean on the PR by rebasing this change onto a previous commit